TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -22,7 +22,7 @@ import Foundation
 ///:nodoc:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0575bf93ed2771bbac49a8fb73c7e54f5a5940f2/TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift#L22"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>SecureStorage.swift:22</code></td>
    </tr>
  </tbody>
</table>